README.md

```diff
@@ -1,54 +1,104 @@
-KΛTEGORY
-========
 [![Download](https://api.bintray.com/packages/kategory/maven/kategory/images/download.svg)](https://bintray.com/kategory/maven/kategory/_latestVersion)
 [![Build Status](https://travis-ci.org/kategory/kategory.svg?branch=master)](https://travis-ci.org/kategory/kategory/)
 [![Kotlin version badge](https://img.shields.io/badge/kotlin-1.1.51-blue.svg)](http://kotlinlang.org/)
 [![Hex.pm](https://img.shields.io/hexpm/l/plug.svg)](http://www.apache.org/licenses/LICENSE-2.0)
-[![codecov](https://codecov.io/gh/kategory/kategory/branch/master/graph/badge.svg)](https://codecov.io/gh/kategory/kategory)
 
-Functional Datatypes and abstractions for Kotlin inspired by [Cats][cats]
+KΛTEGORY is a library for Typed Functional Programming in Kotlin.
+It includes the most popular data types, type classes and abstractions such as `Functor`, `Applicative`, `Monad` and friends empowering users to define pure FP apps and libraries built atop higher order abstractions. Use the below list to learn more about KΛTEGORY's main features.
 
-<img height="100" src="https://avatars2.githubusercontent.com/u/29458023?v=4&amp;s=200" width="100">
+- [Documentation](http://kategory.io)
+- [Patterns](/docs/patterns/error_handling/)
+- [Type classes](http://kategory.io/docs/typeclasses/functor/)
+- [Data types](http://kategory.io/docs/datatypes/option/)
+- [Effects](http://kategory.io/docs/effects/io/)
+- [Optics](http://kategory.io/docs/optics/iso/)
 
 # Add it to your project
 
-Use it at your own risk, the actual state is not production ready.
-
 Add it in your root `build.gradle` at the end of repositories.
 
 ```groovy
 allprojects {
     repositories {
         jcenter()
         maven { url 'https://kotlin.bintray.com/kotlinx' }
-        maven { url "http://dl.bintray.com/kotlin/kotlin-dev" }
+        maven { url 'https://dl.bintray.com/kategory/maven/' }
     }
 }
 ```
 
-Add the dependency into project `build.gradle`
+Add the dependencies into the project's `build.gradle`
 
 ```groovy
+apply plugin: 'kotlin-kapt' //optional
+apply from: rootProject.file('gradle/generated-kotlin-sources.gradle') //optional
```

Comment:
Client projects won't have this file. If someone blindly copies the config (which they will), this will fail. Perhaps we should explain why we do this and point to the raw file in our repo.